packages/format-library/src/text-color/inline.js

```diff
@@ -34,7 +34,7 @@ export function getActiveColor( formatName, formatValue, colors ) {
 	}
 	const currentClass = activeColorFormat.attributes.class;
 	if ( currentClass ) {
-		const colorSlug = currentClass.replace( /.*has-(.*?)-color.*/, '$1' );
+		const colorSlug = currentClass.replace( /.*has-(.*?)-color$/, '$1' );
```

Comment:
Hi @abaicus, thank you for reporting and proposing a fix to this bug 👍
There are no guarantees that the class attribute will only contain the color class. It may contain another class after "has-test-color-color another-class". What if instead we use a greedy regex and exclude the spaces. I guess something like `/.*has-([^\s]*)-color.*/` would work very well for this case.

Reply:
Hi @jorgefilipecosta, thanks for taking the time to review this :+1: 

Yes, it seems that a greedy regex with excluded spaces would be the better way to go indeed. 

I will submit a commit to change it. :+1:

Thank you! :see_no_evil: